src/entities/alerts/index.ts

```diff
@@ -0,0 +1,45 @@
+/** @format */
+import { GlobalConfig } from "../../config";
+import Network, { Models } from "../../network";
+import { IPagedEntity } from "../../models";
+
+import { IAlert, IAlertSummary } from "./models";
+
+export class Alerts {
+  baseUrl: string = "alerts";
+
+  /**
+   * Retrieves an alert matching the given identifier
```

Comment:
Love that you're adding these in! maybe we should look to implement something like JSdoc early to ensure we implement a good standard going forward? 

https://jsdoc.app/
https://alligator.io/js/jsdoc/

Reply:
Added 👍